ci/test/00_setup_env_native_asan.sh

```diff
@@ -6,7 +6,7 @@
 
 export LC_ALL=C.UTF-8
 
-export CI_IMAGE_NAME_TAG="docker.io/ubuntu:24.04"
+export CI_IMAGE_NAME_TAG="mirror.gcr.io/ubuntu:24.04"
```

Comment:
the ASAN job (and others) currently runs on a GitHub hosted runner and these are exempt from the rate-limiting: https://github.com/actions/runner-images/issues/1445#issuecomment-714832439